### PR TITLE
Alert dialog when reset is clicked in app bar buttons

### PR DESCRIPTION
**Pull request by LCB2021029-Badri**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by pratyaksh1610** [comment]
Tests are failing.

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 1]
```diff
@@ -235,4 +235,11 @@
         comment="Generic message about an unknown error, to be shown in snackbars and such"
         translatable="false"
         >@string/webview_crash_unknown</string>
+
+    <!-- for alert dialog in appearances   -->
+    <string name="alert_title">Confirm Reset</string>
+    <string name="alert_message">Are you sure you want to reset all custom toolbar buttons?</string>
+    <string name="alert_positive">Yes</string>
```
Comment (pratyaksh1610) [inline AnkiDroid/src/main/res/values/01-core.xml:242]: I think `Yes` and `No` strings are already there and need not be re-declared.

Comment (LCB2021029-Badri) [reply AnkiDroid/src/main/res/values/01-core.xml]: ok, i will try using them

**Comment by LCB2021029-Badri** [comment]
The code has passed all the checks. @pratyaksh1610

**Comment by LCB2021029-Badri** [comment]
can anyone approve this, if there is no error :)

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt [hunk 2]
```diff
@@ -32,32 +33,42 @@ class CustomButtonsSettingsFragment : SettingsFragment() {
 
     override fun initSubscreen() {
         // Reset toolbar button customizations
+
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt:36]: unnecessary blank line

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt [hunk 3]
```diff
@@ -32,32 +33,42 @@ class CustomButtonsSettingsFragment : SettingsFragment() {
...
+                        remove("customButtonBury")
+                        remove("customButtonSuspend")
+                        remove("customButtonFlag")
+                        remove("customButtonDelete")
+                        remove("customButtonEnableWhiteboard")
+                        remove("customButtonSaveWhiteboard")
+                        remove("customButtonWhiteboardPenColor")
+                        remove("customButtonClearWhiteboard")
+                        remove("customButtonShowHideWhiteboard")
+                    }
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt:65]: nit: could you extract this into a method to improve the readability?

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 4]
```diff
@@ -235,4 +235,9 @@
         comment="Generic message about an unknown error, to be shown in snackbars and such"
         translatable="false"
         >@string/webview_crash_unknown</string>
+
+    <!-- for alert dialog in appearances   -->
+    <string name="alert_title">Confirm Reset</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/01-core.xml:240]: unnecessary

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt [hunk 5]
```diff
@@ -32,32 +33,42 @@ class CustomButtonsSettingsFragment : SettingsFragment() {
...
-                remove("customButtonWhiteboardPenColor")
-                remove("customButtonClearWhiteboard")
-                remove("customButtonShowHideWhiteboard")
-            }
-            // #9263: refresh the screen to display the changes
-            refreshScreen()
+            val builder = AlertDialog.Builder(requireContext())
+            builder.setTitle(R.string.alert_title)
+                .setMessage(R.string.alert_message)
+                .setPositiveButton(R.string.dialog_yes) { _, _ ->
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt:42]: Yes or No dialogs usually aren't good. I suggest something like `Reset custom buttons` → `OK` / `Cancel`

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 6]
```diff
@@ -235,4 +235,9 @@
...
+    <string name="alert_message">Are you sure you want to reset all custom toolbar buttons?</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/01-core.xml:241]: `alert_title` and `alert_message` are generic keys. As those strings are actually specific to custom buttons, the keys should be more specific as well

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

**Pull request closed without merging**